dll/win32/shimgvw/shimgvw.c

```diff
@@ -959,6 +977,58 @@ ImageView_OnSize(HWND hwnd, UINT state, INT cx, INT cy)
     }
 }
 
+static LRESULT CALLBACK
```

Comment:
```suggestion
static LRESULT
```
This is not callback.

Reply:
Why? Since `ImageView_WndProc` calls it as `return ImageView_Delete(hwnd);` (and `ImageView_Modify`) is the same, the compiler can generate better code if that function is also `__stdcall`.